extensions/google-translate/src/translate.tsx

```diff
@@ -30,6 +30,12 @@ export default function Command(): ReactElement {
     }>();
 
     const promises = Promise.all([
+      translate(toTranslate, {
+        to: preferences.lang1,
+      }),
+      translate(toTranslate, {
+        to: preferences.lang2,
+      }),
```

Comment:
This is too much calls, it will cause 429 (Too many requests) very fast and translate will not work for some time